pytensor/link/pytorch/dispatch/basic.py

```diff
@@ -150,6 +151,19 @@ def makevector(*x):
     return makevector
 
 
+@pytorch_funcify.register(IfElse)
+def pytorch_funcify_IfElse(op, **kwargs):
+    n_outs = op.n_outs
+
+    def ifelse(cond, *true_and_false, n_outs=n_outs):
+        if cond:
+            return torch.stack(true_and_false[:n_outs])
```

Comment:
We don't need to stack, and shouldn't, because outputs can have different dimensions / sizes